pydeseq2/dds.py

```diff
@@ -227,14 +230,34 @@ def deseq2(self) -> None:
             # for genes that had outliers replaced
             self.refit()
 
-    def fit_size_factors(self) -> None:
+    def fit_size_factors(
+        self, fit_type: Literal["ratio", "iterative"] = "ratio"
+    ) -> None:
         """Fit sample-wise deseq2 normalization (size) factors.
 
-        Uses the median-of-ratios method: see :func:`pydeseq2.preprocessing.deseq2_norm`.
+        Uses the median-of-ratios method: see :func:`pydeseq2.preprocessing.deseq2_norm`,
+        unless each gene has at least one zero, in which case it switches to
```

Comment:
```suggestion
        unless each gene has at least one zero, in which case it switches to
```
do you mean zero gene count in at least one of the samples? Could you make it clear?

Reply:
I've updated the docstring to make it clearer